Mathlib/RingTheory/Coprime/Basic.lean

```diff
@@ -367,6 +367,29 @@ theorem neg_neg {x y : R} (h : IsCoprime x y) : IsCoprime (-x) (-y) :=
 theorem neg_neg_iff (x y : R) : IsCoprime (-x) (-y) ↔ IsCoprime x y :=
   (neg_left_iff _ _).trans (neg_right_iff _ _)
 
+section abs
+
+variable [LinearOrder R] [AddLeftMono R]
+
+lemma abs_left_iff (x y : R) : IsCoprime |x| y ↔ IsCoprime x y := by
```

Comment:
I would consider making this (and `abs_right_iff`) `@[simp]` lemmas but it seems the rest of the file doesn't do that either. So I suppose we could also keep it as is.

Reply:
I was copying the `neg` lemmas here. I think I'd prefer to leave them without `simp` and leaving song the attribute for someone to do across the whole file.